data/data/manifests/bootkube/openshift-etcd-service.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: openshift-etcd
```

Comment:
`temporary-etcd`  We need this to be a valid service that we can use in our operators.  Once our operators switch, we make the "normal" etcd into a copy of this, switch the operators to that, and this etcd goes away.